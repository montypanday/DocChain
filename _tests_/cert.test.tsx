import * as React from "react";
import { CertDrawer } from "../clientapp/components/explorers/certdrawer";
import { CertPop } from "../clientapp/components/explorers/certpop";
import * as renderer from "react-test-renderer";
import * as ReactTestUtils from 'react-dom/test-utils';
import * as ReactDOM from 'react-dom'

const init = () => {

    const props = {
        showCert: false,
    }

    const component = renderer.create(
        <div>
            <CertDrawer {...props}></CertDrawer>
            <CertPop></CertPop>
        </div>,
    );

    return component;
}

describe("Cert changes the class when hovered", () => {
    const certGroup = init();

    it('Clicked', () => {
        ReactTestUtils.Simulate.click(certGroup);
        let tree = certGroup.toJSON();
        expect(tree).toMatchSnapshot();
    });

});