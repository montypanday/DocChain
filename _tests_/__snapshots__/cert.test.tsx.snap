// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cert changes the class when hovered Clicked 1`] = `
<div>
  <button
    className="history action-btn btn-line btn btn-default"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    type="button"
  >
     
    <i
      className="fa fa-history"
    />
    
  </button>
  <div
    className="popover right"
    id="popover-trigger-click-root-close"
    role="tooltip"
    style={
      Object {
        "display": "block",
        "left": undefined,
        "top": undefined,
      }
    }
  >
    <div
      className="arrow"
      style={
        Object {
          "left": undefined,
          "top": undefined,
        }
      }
    />
    <h3
      className="popover-title"
    >
      File Certificate
    </h3>
    <div
      className="popover-content"
    >
      Here is where certificate data goes
    </div>
  </div>
</div>
`;
